app/controllers/users.js

```diff
@@ -30,3 +31,12 @@ exports.getUsers = (req, res, next) => {
     .then(users => res.send(users))
     .catch(error => next(error));
 };
+
+exports.getUserAlbums = (req, res, next) => {
+  req.userToken = jwt.validate(req.headers.authorization);
```

Comment:
This is not the responsibility of this controller, you should do these things in middlewares